packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -75,20 +82,53 @@ class IsometricTileMapComponent extends PositionComponent {
   /// This is the size the tiles will be drawn (either original or overwritten).
   Vector2 get effectiveTileSize => destTileSize ?? tileset.srcSize;
 
-  /// This is the vertical height of each block; by default it's half the tile
-  /// size.
+  /// The current scaling factor for the isometric view.
+  double get scalingFactor => effectiveTileSize.y / effectiveTileSize.x;
+
+  /// This is the vertical height of each block; by default it's half the tile size.
   double get effectiveTileHeight => tileHeight ?? (effectiveTileSize.y / 2);
 
+  /// gets the effective propSize. PropSize should be assigned if prop dimension are different from srcSize of tilset
+  Vector2 get propsMaxSize => propSize ?? effectiveTileSize;
+
+  // render specific variables
```

Comment:
Is this like a region? If so we should use a region comment

Reply:
I just like to categorize variables ... 🤦🏽‍♂️ that just leaked into this code, I will remove it as this is a different style compared to rest.